pmc/journal_recommendation/journal_recommendation_quick_fix.py

```diff
@@ -0,0 +1,221 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.9.1+dev
+#   kernelspec:
+#     display_name: Python [conda env:annorxiver]
+#     language: python
+#     name: conda-env-annorxiver-py
+# ---
+
+# # Re-Run KNearestNeighbors for Journal Recommendation
+
+# This notebook is designed to predict journals based on an updated version of document vector generation. Before I was doing a simple token analysis using spaces, but now I incorporated Spacy with lemma generation. To simplify running the recommendation notebook all over again, I'm just using the 300 dimensions to train a KNN-model and to compare its performance against a random baseline.
```

Comment:
What do you mean "by spaces" here?

Reply:
Spaces here meant whitespace: e.g. the space between the words in this sentence. I recognize that description was a little vague and updated the notebook to be more explicit.